pom.xml

```diff
@@ -39,7 +39,7 @@
     </scm>
 
     <properties>
-        <jenkins.version>2.73.3</jenkins.version>
+        <jenkins.version>2.164.1</jenkins.version>
```

Comment:
I remember there was a reason to set this specific number here. This version means the first Jenkins version we are compatible with.
@chicofaccin any thoughts?